labels.json

```diff
@@ -0,0 +1,39 @@
+
+[
+  { "name": "Package: analyzer", "color": "#f9df95" },
+  { "name": "Package: build", "color": "#f9df95" },
+  { "name": "Package: bundler", "color": "#f9df95" },
+  { "name": "Package: cli", "color": "#f9df95" },
+  { "name": "Package: project-config", "color": "#f9df95" },
+  { "name": "Package: linter", "color": "#f9df95" },
+  { "name": "Browser: Chrome", "color": "#3477c9" },
+  { "name": "Browser: Safari", "color": "#3477c9" },
+  { "name": "Browser: Firefox", "color": "#3477c9" },
+  { "name": "Browser: Edge", "color": "#3477c9" },
+  { "name": "Browser: IE", "color": "#3477c9" },
+  { "name": "Browser: Other", "color": "#3477c9" },
+  { "name": "Browser: Safari", "color": "#3477c9" },
+  { "name": "Platform: Linux", "color": "#bfdadc" },
+  { "name": "Platform: macOS", "color": "#bfdadc" },
+  { "name": "Platform: Windows", "color": "#bfdadc" },
+  { "name": "Platform: Android", "color": "#bfdadc" },
+  { "name": "Platform: iOS", "color": "#bfdadc" },
+  { "name": "Status: Abandoned", "color": "#000000" },
+  { "name": "Status: Accepted", "color": "#009800" },
+  { "name": "Status: Available", "color": "#bfe5bf" },
+  { "name": "Status: Blocked", "color": "#e11d21" },
+  { "name": "Status: Completed", "color": "#006b75" },
+  { "name": "Status: In Progress", "color": "#cccccc" },
+  { "name": "Status: On Hold", "color": "#e11d21" },
+  { "name": "Status: Pending", "color": "#fef2c0" },
+  { "name": "Status: Review Needed", "color": "#fbca04" },
+  { "name": "Status: Revision Needed", "color": "#e11d21" },
```

Comment:
Are Review Needed and Revision Needed necessary? They seem like properties of a Pull Request that are already modeled in a first class way by GitHub